assets/css/quick-filter.css

```diff
@@ -0,0 +1,54 @@
+.quick-filter {
+  border-radius: 0.5rem;
+  border: 1px solid var(--neutral-300, #d0d5dd);
+  background: var(--white, #fff);
+  box-shadow: 0px 1px 2px 0px rgba(16, 24, 40, 0.15);
+  display: inline-flex;
+  padding: 0.5rem;
+  align-items: center;
+  gap: 0.25rem;
+  cursor: pointer;
+}
+
+.quick-filter:last-of-type {
+  margin-right: 16px;
+}
+
+.quick-filter.quick-filter__name {
+  color: var(--neutral-500);
+}
+
+.quick-filter .quick-filter__icon {
+  background-color: var(--neutral-500);
+  width: 1.25rem;
+  height: 1.25rem;
+}
+
+.quick-filter.selected {
+  border-radius: 0.5rem;
+  background: var(--primary-900, #0f1815) !important;
```

Comment:
This is not needed the hover takes priority, we need different styles for .selected:hover, and we need to ask Design for those styles.
In the meantime, please remove the !important

Reply:
Hey, Without the `!important` property the selected styles aren't getting applied immediately after click event. You can check this in a mobile phone and notice the difference without `!important`